circle.yml

```diff
@@ -53,6 +53,7 @@ dependencies:
 
 test:
   override:
+    - make test-doc
```

Comment:
It doesn't on my system, and I don't see it happen on CircleCI logs I looked at. Where is it happening?